app/design/frontend/base/default/template/ebanx/form/baloto.phtml

```diff
@@ -1,13 +1,20 @@
 <?php
     $skin = Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_SKIN);
+    $showInlineIcon = Mage::helper('ebanx')->hasToShowInlineIcon();
+    $icon = '<div><img src="' . $skin . 'frontend/base/default/ebanx/ebanx-baloto.png" width="40" /></div>';
 ?>
 
+<?php if($showInlineIcon) {
+	echo $icon;
+} ?>
+
 <ul class="form-list ebanx-payment-method ebanx-es-method" id="payment_form_<?php echo $this->getMethodCode() ?>" style="display:none;">
     <li>
         <div class="<?php echo $this->getMethodCode() ?>-instructions-content">
-            <div>
-                <img src="<?= $skin . 'frontend/base/default/ebanx/ebanx-baloto.png' ?>" width="40" />
-            </div>
+
+            <?php if(!$showInlineIcon) {
+				echo $icon;
+			} ?>
```

Comment:
The indentation on lines 17 and 16 aren't the same as the indentation on line 15.